bench/bench._test.go

```diff
@@ -0,0 +1,69 @@
+package bench
```

Comment:
is the filename intentionally `bench._test.go`?

Reply:
ooops, not intentional 😄  Thanks1